### PR TITLE
removing keyline

### PR DESCRIPTION
**Pull request by melanierogan**
## Who is this for?

Users of the exhibition guides, for https://github.com/wellcomecollection/wellcomecollection.org/issues/8481

## What is it doing for them?

@alicerichmond has flagged that the grey lines are not part of the design. They were visible in firefox but I could also see them in chrome.

Before:
Screenshot 2022-10-04 at 10 05 33

After:
Screenshot 2022-10-04 at 10 05 52

### CODE REVIEW

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ const ContextTitle = styled(Space).attrs<{ level: number }>(props => ({
   className: font('wb', 3),
   v: { size: 'm', properties: ['margin-bottom'] },
 }))<{ level: number }>``;
+console.log(ContextTitle, 'this is the output of the styling');
```
Comment (rcantin-w) [inline content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:54]: Do you want to keep this?

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx [hunk 2]
```diff
@@ -214,7 +215,7 @@ const Stop: FC<{
                 properties: ['margin-bottom'],
               }}
             >
-              <Divider color="warmNeutral.400" isKeyline={true} />
+              <Divider color="warmNeutral.400" isKeyline={false} />
```
Comment (rcantin-w) [inline content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:218]: Is it worth keeping this if we're not displaying it?

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx (hunk 1 above)
Comment (melanierogan) [reply content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:54]: oops! Rookie error, get out of here console log

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx (hunk 2 above)
Comment (davidpmccormick) [reply content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:217]: agree – just delete the line?

**Review by davidpmccormick — APPROVED** [review]
🔑

File: content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx [hunk 3]
```diff
@@ -214,7 +214,7 @@ const Stop: FC<{
                 properties: ['margin-bottom'],
               }}
             >
-              <Divider color="warmNeutral.400" isKeyline={true} />
+              <Divider color="warmNeutral.400" />
```
Comment (rcantin-w) [inline content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:217]: I think David meant to whole thing, it won't display without `isKeyline` or `isThin`, right? Maybe the required margin can be dealt with differently?

Comment (davidpmccormick) [reply content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx:217]: You're right – that is what I meant

Comment (melanierogan) [reply content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx]: Ok, it is completely gone now

**Review by rcantin-w — COMMENTED** [review]
Could this spacing be moved to the sibling as a conditional `margin-top` instead of having an empty div?

```
 {!isFirstStop && (
            <Space
              v={{
                size: 'xl',
                properties: ['margin-bottom'],
              }}
            >
            </Space>
          )}
```

**Pull request merged**